plugins/boltz/normal_fsm.go

```diff
@@ -0,0 +1,306 @@
+package boltz
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"time"
+
+	"github.com/BoltzExchange/boltz-lnd/boltz"
+	"github.com/bolt-observer/agent/lightning"
+	"github.com/btcsuite/btcd/chaincfg"
+)
+
+const (
+	BtcPair = "BTC/BTC"
+	Btc     = "BTC"
+)
+
+// Normal (submarine) swap finite state machine
+
+func FsmInitialNormal(in FsmIn) FsmOut {
```

Comment:
Lets try to avoid `normal` terminology and stickt to swap-in / out or submarin/reverse-submarine